components/ErrorBanner/ErrorBanner.tsx

```diff
@@ -12,27 +13,52 @@ import {
 } from "./styles";
 
 export function ErrorBanner({ errorOrigin }: { errorOrigin?: ErrorOriginT }) {
+  const { isWrongChain } = useWalletContext();
   const { errorMessages, removeErrorMessage } = useErrorContext(errorOrigin);
 
-  if (!errorMessages.length) return null;
+  if (!errorMessages.length && !isWrongChain) return null;
 
   return (
     <Wrapper>
-      {errorMessages.map((message) => (
-        <ErrorMessageWrapper key={message?.toString()}>
+      {isWrongChain ? (
+        <Error message={wrongChainMessage} />
```

Comment:
No `removeErrorMessage` for this one, so the "close" button doesn't show. I think this makes sense — the only way to get rid of this error is to switch to the correct chain